### PR TITLE
Issue #sb-20425 fix: Too many messages in browser console log

### PR DESCRIPTION
**Pull request by VenkatKadiveti**

### CODE REVIEW

File: src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts [hunk 1]
```diff
@@ -569,8 +567,6 @@ export class QuestionListComponent implements OnInit, AfterViewInit, OnDestroy {
         this.goToNextQuestionStatus = true;
         this.fetchQuestionList();
       }
-    }, error => {
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:572]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts [hunk 2]
```diff
@@ -640,8 +636,6 @@ export class QuestionListComponent implements OnInit, AfterViewInit, OnDestroy {
     return this.actionService.post(request).pipe(map((response) => {
       this.questionList.push(_.assign(request.data.request.assessment_item.metadata, {'identifier': _.get(response, 'result.node_id')}));
       return response;
-    }, err => {
-      console.log(err);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:644]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts [hunk 3]
```diff
@@ -766,8 +759,6 @@ export class QuestionListComponent implements OnInit, AfterViewInit, OnDestroy {
     const reqBody = requestBody;
     return this.itemsetService.updateItemset(reqBody, identifier).pipe(map((response) => {
       return response;
-    }, err => {
-      console.log(err);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:770]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts [hunk 4]
```diff
@@ -778,8 +769,6 @@ export class QuestionListComponent implements OnInit, AfterViewInit, OnDestroy {
     return this.helperService.updateContent(reqBody, contentId).pipe(map((response) => {
       this.existingContentVersionKey = _.get(response, 'result.versionKey');
       return response;
-    }, err => {
-      console.log(err);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/cbse-program/components/question-list/question-list.component.ts:782]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/cbse-program/components/question-preview/question-preview.component.ts [hunk 5]
```diff
@@ -27,8 +27,6 @@ export class QuestionPreviewComponent implements OnInit, OnChanges {
         this.theme = theme;
         const context = this.getContext();
         this.playerConfig =  this.setPlayerConfig(context, theme);
-      }, error => {
-        console.log(error);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/cbse-program/components/question-preview/question-preview.component.ts:31]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/cbse-program/components/question-preview/question-preview.component.ts [hunk 6]
```diff
@@ -45,8 +43,6 @@ export class QuestionPreviewComponent implements OnInit, OnChanges {
           this.theme = theme;
           const context = this.getContext();
           this.playerConfig =  this.setPlayerConfig(context, theme);
-        }, error => {
-          console.log(error);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/cbse-program/components/question-preview/question-preview.component.ts:49]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts [hunk 7]
```diff
@@ -144,8 +144,6 @@ export class DataChartComponent implements OnInit, OnDestroy {
           }
           this.getDataSetValue(res);
 
-        }, (err) => {
-          console.log(err);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts:148]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts [hunk 8]
```diff
@@ -249,8 +247,6 @@ export class DataChartComponent implements OnInit, OnDestroy {
         const currentFilterValue = _.get(this.filtersFormGroup, 'value');
         this.filtersFormGroup.patchValue(currentFilterValue);
       }
-    }, err => {
-      console.log('failed to update chart data', err);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/dashboard/components/data-chart/data-chart.component.ts:253]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/dashboard/components/usage-reports/usage-reports.component.ts [hunk 9]
```diff
@@ -124,10 +123,9 @@ export class UsageReportsComponent implements OnInit, AfterViewInit {
             this.renderFiles({}, data);
           }
         } else {
-          console.log(response);
         }
 
-      }, err => { console.log(err); });
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/dashboard/components/usage-reports/usage-reports.component.ts:130]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/groups/components/add-member/add-member.component.ts [hunk 10]
```diff
@@ -75,7 +75,7 @@ export class AddMemberComponent implements OnInit, OnDestroy {
           this.googleCaptchaSiteKey = captchaConfig.key || '';
           this.isCaptchEnabled = captchaConfig.isEnabled || false;
         } catch (e) {
-          console.log(_.get(res, 'result.response'));
+           console.log(_.get(res, 'result.response'));
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/groups/components/add-member/add-member.component.ts:78]: console log not deleted.

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts [hunk 11]
```diff
@@ -294,8 +294,7 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
       .subscribe(res => {
         this.contentStatus = res.content || [];
         this.calculateProgress();
-      },
-        err => console.log(err, 'content read api failed'));
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts:298]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts [hunk 12]
```diff
@@ -293,9 +293,6 @@ export class UserOrgManagementComponent implements OnInit, AfterViewInit, OnDest
           'accounts_unclaimed': result['accounts_unclaimed'] ? result['accounts_unclaimed'] : 0,
           'accounts_eligible': result['accounts_eligible'] ? result['accounts_eligible'] : 0
         };
-      },
-      error => {
-        console.log(error);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts:298]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts [hunk 13]
```diff
@@ -309,9 +306,6 @@ export class UserOrgManagementComponent implements OnInit, AfterViewInit, OnDest
           'blocks': result['blocks'] ? result['blocks'] : 0,
           'schools': result['schools'] ? result['schools'] : 0
         };
-      },
-      error => {
-        console.log(error);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts:314]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts [hunk 14]
```diff
@@ -327,9 +321,6 @@ export class UserOrgManagementComponent implements OnInit, AfterViewInit, OnDest
           'subOrgRegistered': result['subOrgRegistered'] ? result['subOrgRegistered'] : 0,
           'rootOrgRegistered': result['rootOrgRegistered'] ? result['rootOrgRegistered'] : 0
         };
-      },
-      error => {
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts:331]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts [hunk 15]
```diff
@@ -339,9 +330,6 @@ export class UserOrgManagementComponent implements OnInit, AfterViewInit, OnDest
       data => {
         const result = _.get(data, 'result');
         this.geoSummary = result;
-      },
-      error => {
-        console.log(error);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts:344]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts [hunk 16]
```diff
@@ -351,9 +339,6 @@ export class UserOrgManagementComponent implements OnInit, AfterViewInit, OnDest
       data => {
         const result = _.get(data, 'result');
         this.validatedUserSummary = result;
-      },
-      error => {
-        console.log(error);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts:356]: Use error block and toasterService to show the appropriate message.

File: src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts [hunk 17]
```diff
@@ -456,9 +441,6 @@ export class UserOrgManagementComponent implements OnInit, AfterViewInit, OnDest
         response => {
           const url = (_.get(response, 'result.signedUrl'));
           if (url) { window.open(url, '_blank'); }
-        },
-        error => {
-          console.log(error);
```
Comment (rajeshkumaravel) [inline src/app/client/src/app/modules/manage/components/user-org-management/user-org-management.component.ts:461]: Use error block and toasterService to show the appropriate message.

**Review by rajeshkumaravel — CHANGES_REQUESTED** [review]

**Review by rajeshkumaravel — APPROVED** [review]

**Comment by VenkatKadiveti** [comment]
Created new PR

**Pull request closed without merging**